content/ja/handbook/engineering/architecture/design-documents/ai_gateway/index.md

```
@@ -0,0 +1,495 @@
+</tbody>
+</table>
+</div>
+
+
+## 概要
+
+AI Gateway は、利用しているインスタンス（self-managed、Dedicated、GitLab.com）にかかわらず、すべての GitLab ユーザーに AI 機能を提供するスタンドアロンサービスです。
+
+当初、すべての AI Gateway デプロイは GitLab（組織）が管理しており、GitLab.com とすべての GitLab self-managed インスタンスが同じ Gateway を使用していました。GitLab Duo Self-Hosted の導入により、[お客様も自身の環境に AI Gateway のインスタンスをデプロイできる](https://docs.gitlab.com/install/install_ai_gateway/) ようになりました。
+
+AI Gateway は、グローバルにアクセス可能な `cloud.gitlab.com/ai/*` ルートからルーティングされるトラフィックを処理する API Gateway です。IDE は現在、GitLab Rails インスタンスを介して間接的に `cloud.gitlab.com/ai/*` を使用しています。将来的には、IDE などのネットワークエンドポイントが `cloud.gitlab.com` に直接接続できるようにする予定です（詳細は [直接接続クライアント](#directly-connected-clients) を参照）。AI Gateway はその後、このトラフィックを他のサービス、つまり AI プロバイダーとそのモデルに転送します。この north/south トラフィックパターンにより、どのリクエストをどこに送信するかを制御し、必要に応じてリダイレクトされたリクエストの内容を変換できます。
```

markdownlint-cli2 (0.22.1)

[warning] Link fragments should be valid

(MD051, link-fragments)